### ISSUE TITLE
RFC: Deprecate in favor of ember-auto-import?

### ISSUE THREAD

**Issue by jrjohnson** [elwayman02/ember-cli-password-strength#72]
I believe that this addon can no be replaced with the installation of `ember-auto-import` which can import and run directly from node_modules.  It also supports async imports so 
```
  passwordStrengthScore: computed('password', async function () {
    const { default: zxcvbn } = await import('zxcvbn');
    const obj = zxcvbn(password);
    return obj.score;
  }),

```
Works!

Wanted to open up this issue for input before going that route.

**Comment by elwayman02** [comment]
That seems like a reasonable approach. Maybe the resolution here is to update the README to suggest that solution for versions of Ember that ember-auto-import supports, and then marking this addon as supporting older versions if reasonable?

**Comment by elwayman02** [comment]
Closed by #74
